examples/xml/behat.yml

```diff
@@ -0,0 +1,18 @@
+default:
+  extensions:
+    Arachne\ServiceContainer\ArachneExtension:
+      base_url: http://urlecho.appspot.com
+      paths:
+        schema_file_dir: %paths.base%/schemas
+        request_file_dir: %paths.base%/requests
+        response_file_dir: %paths.base%/responses
+      auth:
+        provider: Arachne\Auth\DummyProvider
+      headers:
+        Authorization: Token token=123456
+  suites:
+    json:
```

Comment:
This is my OCD kicking in, since it is an xml config file, can you change the suite name :)